MASTER FUNCTIONS LIBRARY.vbs

```diff
@@ -7926,95 +7926,245 @@ function MAXIS_dialog_navigation()
 			End if
 			transmit
 		End if
-	End if
+    End if
+
+'This part takes care of remaining navigation buttons, designed to go to a single panel.
+    'APPL
+    If ButtonPressed = APPL_button then call navigate_to_MAXIS_screen("APPL", "") 'Application (APPL)
+    If ButtonPressed = MEMB_button then call navigate_to_MAXIS_screen("MEMB", "") 'Household Members (MEMB)
+    If ButtonPressed = MEMI_button then call navigate_to_MAXIS_screen("MEMI", "") 'Additional Member Info (MEMI)
```

Comment:
These navigations are tied to STAT, so they will not work. They also exist in the STAT section of the function's code. Please remove.